src/Assignments/SimpleProblemPage.tsx

```diff
@@ -530,9 +536,13 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
                                 setOpenDrawer={_.isNil(selectedGradeId) ? undefined : setOpenDrawer}
                             />
                             {/* Temporarily disabled for release.  */}
```

Comment:
Should probably update this comment to explain the `university === 5` to the OSS community.